WizardsDen/Children/Frigid/README.md

```diff
@@ -0,0 +1,20 @@
+# Frigid
+
+<img src="frigid-logo.png" width=300>
```

Comment:
rename file to just `logo.png`

Reply:
> archive https://421034.xyz/ if it is not already done

recycling the same domain for frigid (unless i misread it)

Reply:
what why did that go in the review not comments

i hate github mobile